xml/storage_multipath.xml

```diff
@@ -444,165 +410,165 @@
+    service <filename>multipathd.service</filename> will be disabled. The
+    system will be configured as described in
+    <xref linkend="vl-multipath-planning-type-noroot-noinitrd"/> in
+    <xref linkend="sec-multipath-planning-type-noroot"/>.
+    Before adding SAN hardware, you will need to enable and start
+    <filename>multipathd.service</filename>. We recommend creating a
+    <literal>blacklist</literal> entry in the
+    <filename>/etc/multipath.conf</filename> for the root device (see
+    <xref linkend="sec-multipath-blacklist"/>).
+   </para>
   </sect2>
+
   <sect2 xml:id="sec-multipath-installing-mp">
-    <title>Installing with connected multipath devices</title>
-    <para>
-      If multipath devices are connected to the system at installation time,
-      &yast; will detect this and display a pop-up window asking the user
-      whether multipath should be
-      enabled before entering the partitioning stage.
-    </para>
-    <informalfigure>
-     <mediaobject>
-      <textobject role="description"><phrase>
+   <title>Installing with connected multipath devices</title>
+   <para>
+    If multipath devices are connected to the system at installation time,
+    &yast; will detect them and display a pop-up window asking you whether
+    multipath should be enabled before entering the partitioning stage.
+   </para>
+   <informalfigure>
+    <mediaobject>
+     <textobject role="description"><phrase>
           &yast; multipath dialog</phrase>
-      </textobject>
-      <imageobject role="fo">
-       <imagedata format="PNG" fileref="multipath-popup.png" width="50%"/>
-      </imageobject>
-      <imageobject role="html">
-       <imagedata fileref="multipath-popup.png" width="50%" format="PNG"/>
-      </imageobject>
-     </mediaobject>
-    </informalfigure>
-    <para>
-      If the user selects
-      &ldquo;No&rdquo; at this prompt (not recommended), installation will
-      proceed as in <xref linkend="sec-multipath-installing-nomp"/>.
-      In the partitioning stage, you should <emphasis>avoid
-      touching devices</emphasis> that will later be part of a multipath map.
-    </para>
+     </textobject>
+     <imageobject role="fo">
+      <imagedata format="PNG" fileref="multipath-popup.png" width="50%"/>
+     </imageobject>
+     <imageobject role="html">
+      <imagedata fileref="multipath-popup.png" width="50%" format="PNG"/>
+     </imageobject>
+    </mediaobject>
+   </informalfigure>
+   <para>
+    If you select &ldquo;No&rdquo; at this prompt (not recommended), the
+    installation will proceed as in
+    <xref linkend="sec-multipath-installing-nomp"/>. In the partitioning stage,
+    do not use/edit devices that will later be part of a multipath map.
+   </para>
+   <para>
+    If you select &ldquo;Yes&rdquo; at the multipath prompt,
+    <command>multipathd</command> will run during the installation. No
+    blocklist will be configured, thus all SCSI and DASD devices, including
```

Comment:
I'm not keen on using blocklists here, but we decided to use inclusive language and replace the blacklist with something else, even though in the product it is blacklist. I'm fine with using another word than blocklist if blocklist has a different meaning. And I'm all ears for your suggesstions.

Reply:
Hm, we've been using "blacklist" all over the place in this document (I count 39 occurences in my version). It's even in section headers. I don't see the point in avoiding it here. IMO clarity and non-ambiguity is more important than being politically correct. (I won't hide that I don't understand why anyone would seriously relate this term to skin tone, and yes I've read a couple of discussions on the matter and I'm still not convinced).

If we are serious about the inclusive language, we should replace "blacklist" everywhere, using a consistent term that doesn't look like a typo (e.g. "exclude list"), and prominently explain (in the nomenclature section?) that we use this term instead of the technical one (which isn't going to change over the lifetime of SLE, unless I'm very mistaken). We should also add xrefs everywhere to avoid misunderstanding.

Reply:
Yep, the other blacklist replacement is under construction right now. I agree with you, I'd keep blacklist :) though the decision has been made for me. Well, this particular term is not about the skin tone so much, but about giving colors a meaning or a mood, which can be in different cultures interpreted in a different way. 
I'll go through the chapter (and the doc later) to clarify this change as much as possible.

Reply:
I would be grateful if we could separate this part of the modifications from my work. It's clearly independent.